WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -24,14 +24,13 @@
 
   <p><%= t('about.structure.committees') %>
   <ul>
-    <li><%= t('about.structure.wdc.name_html') %>: <%= format_team_members('wdc') %> <br />
-      <%= t('about.structure.wdc.description') %></li>
-    <li><%= t('about.structure.wrc.name_html') %>: <%= format_team_members('wrc') %> <br />
-      <%= t('about.structure.wrc.description') %></li>
-    <li><%= t('about.structure.results.name_html') %>: <%= format_team_members('results') %> <br />
-      <%= t('about.structure.results.description') %></li>
-    <li><%= t('about.structure.software.name_html') %>: <%= format_team_members('software') %> <br />
-      <%= t('about.structure.software.description') %></li>
+    <% [ "wct", "wdc", "wrc", "results", "software" ].each do |friendly_id| %>
+      <% team = Team.find_by_friendly_id!(friendly_id) %>
```

Comment:
I opted to add an `email` and `rank` columns to the `teams` table, so now we can iterate over the whole table, sorted by `rank` and handle this.

I also chose to remove the ability to create new teams, as @jonatanklosko suggested.